### DISCUSSION TITLE
Possible to provide periodical releases

### DISCUSSION THREAD

**Discussion by ghost** [eteran/edb-debugger#698] [General]
Hello,

I like this project but I don't always have time to find the packaged to compile it. It would be very nice and comfortable to just be able to download and run the binary.

**Comment by eteran** [comment]
I agree, we'll see what we can do :-)

**Comment by ghost** [comment]
Ok. 
I would like to see.  please .

Thanks!

**Comment by ghost** [comment]
Yes that would be nice . because I have broken dev-libs I need to update my O.S which  I mean to do but cannot at this time. Or a compiled version of the program would work fine. 

I think this would help the debugger be more popular in general .

**Comment by eteran** [comment]
I am thinking of developing a quarterly schedule of releases (so long as there has been any development during that quarter).

At the moment, I am looking into using docker to create .deb/.rpm files in as automated a way as possible. Once I figure that out, I'll proceed with a release schedule :-)/

**Comment by ghost** [comment]
Thank you!!! That will make me and many very happy!
